tests/test_multichannel.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+# CREATED:2013-03-08 15:25:18 by Brian McFee <brm2132@columbia.edu>
+#  unit tests for multi-channel functionality
+#
+
+from __future__ import print_function
+
+# Disable cache
+import os
+
+try:
+    os.environ.pop("LIBROSA_CACHE_DIR")
+except:
+    pass
+
+import librosa
+import glob
+import numpy as np
+import scipy.io
+import pytest
+import warnings
+from unittest import mock
+
+
+@pytest.fixture(scope="module", params=["test1_44100.wav"])
+def y_multi(request):
+    infile = request.param
+    return librosa.load(os.path.join("tests", "data", infile),
+                        sr=None, mono=False)
+
+
+def test_stft_multi(y_multi):
+
+    # Verify that a stereo STFT matches on
+    # each channel individually
+    y, sr = y_multi
+
+    D = librosa.stft(y)
+
+    D0 = librosa.stft(y[0])
+    D1 = librosa.stft(y[1])
+
```

Comment:
Trying to go through and make sense of the work being done here. Are we assuming that the test for a stereo file will generalize for files with a larger number of channels?

Reply:
Yes, we are.  The tests so far are minimal, and make use of existing stereo data in the test repo without requiring additional multichannel data.